querytyper.js

```diff
@@ -114,8 +114,10 @@ queryTyperConfig.rootDirs.forEach(rootDir => {
     // Write exports file
     const exportsFileBody =
       generatedQueryNames.length > 0
-        ? generatedQueryNames.map(queryName => `export { ${queryName}, Result as ${queryName}Result, Arguments as ${queryName}Args } from './${queryName}.query';`).join('\n') + '\n'
+        ? generatedQueryNames.map(queryName => `export { ${queryName} } from './${queryName}.query';`).join('\n') + '\n'
         : ''
-    writeIfChanged(path.join(dirItemAbs, queryTyperConfig.exportsFileName), codegenWarning + exportsFileBody)
+    if (exportsFileBody !== '') {
+      writeIfChanged(path.join(dirItemAbs, queryTyperConfig.exportsFileName), exportsFileBody + codegenWarning)
```

Comment:
Files starting with a comment are not allowed with `--isolatedModules`, so I appended the warning at the bottom. :roll_eyes: